README.md

```diff
@@ -1,59 +1,59 @@
 # pi-bashscripts-files
 A collection of Bash scripts and files useful for the Raspberry Pi 4 and other models.
 
-Please note that all of these scripts were tested on a Raspberry Pi 4 running Raspberry Pi OS (32 bit) off of a 16 GB SanDisk MicroSD card and/or a 500 GB Samsung T7 SSD.
+Please note that all of these scripts were tested on a Raspberry Pi 4 running Raspberry Pi OS (32 bit) off of a 16 GB SanDisk MicroSD card and/or a 500 GB Samsung T7 SSD, however not using these products will primarily not affect usage.
 
-Using any other operating system may result in unexpected installation failures or may break your system, so if not using RPiOS 32bit, **be prepared for failures.**
+Using any other operating system may result in unexpected installation failures or may break your system, so if not using RPi-OS 32bit, **be prepared for failures.**
 
-## installation files and such
-I have organized everything into folders. Some folders may have install scripts, some may have packages, or links, or something else. Only use this repository if you know what you're doing.
+## Installation Files and Derivatives
+Everything is organized into folders. Some folders may have install scripts, some may have packages and or links or something else. Only use this repository if you know what you're doing, otherwise you may break your installation of your OS.
 
 
 ## To add to [pi-apps](https://github.com/Botspot/pi-apps/):
 
-These are the scripts i'm working on adding to botspot's Pi apps project. If you would like to help me with them, awesome! just message me: @chunkymilk0309#3309 on discord.
+These are the scripts I'm working on adding to Botspot's Pi apps project. If you would like to help me with them, awesome! just message me: @chunkymilk0309#3309 on Discord.
 
 Alacritty https://github.com/chunky-milk/pi-bashscripts-files/tree/main/alacritty
 
 Bleachbit https://github.com/chunky-milk/pi-bashscripts-files/tree/main/bleachbit
 
-conky manager https://github.com/chunky-milk/pi-bashscripts-files/tree/main/conky-manager
+Conky Manager https://github.com/chunky-milk/pi-bashscripts-files/tree/main/conky-manager
 
-dolphin emu https://github.com/chunky-milk/pi-bashscripts-files/tree/main/dolphin-emu
+Dolphin Emu https://github.com/chunky-milk/pi-bashscripts-files/tree/main/dolphin-emu
 
-dosbox x https://github.com/chunky-milk/pi-bashscripts-files/tree/main/dosbox-x
+Dosbox x https://github.com/chunky-milk/pi-bashscripts-files/tree/main/dosbox-x
 
 Duckstation https://github.com/chunky-milk/pi-bashscripts-files/tree/main/duckstation
 
-freetube https://github.com/chunky-milk/pi-bashscripts-files/tree/main/freetube
+Freetube https://github.com/chunky-milk/pi-bashscripts-files/tree/main/freetube
```

Comment:
FreeTube is already on pi-apps, so this line can be removed.

Reply:
Why are the folder still there, should I remove them?